plaso/storage/file_interface.py

```diff
@@ -895,15 +798,19 @@ def FinalizeTaskStorage(self, task):
     if self._storage_type != definitions.STORAGE_TYPE_SESSION:
       raise IOError('Unsupported storage type.')
 
-    storage_file_path = self._GetTaskStorageFilePath(task)
-    processed_storage_file_path = self._GetProcessedStorageFilePath(task)
+    if not task.storage_format in definitions.STORAGE_FORMATS:
+      raise IOError('Unsupported storage format')
```

Comment:
add "Unsupported storage format" as a reason to raise to the docstring

Reply:
Done